packages/ndla-ui/src/KeyFigure/KeyFigure.tsx

```diff
@@ -7,51 +7,40 @@
  */
 
 import parse from "html-react-parser";
-import styled from "@emotion/styled";
-import { breakpoints, colors, fonts, mq, spacing } from "@ndla/core";
+import { styled } from "@ndla/styled-system/jsx";
 
-const ContentWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  padding: ${spacing.small};
-  align-items: center;
-  ${mq.range({ from: breakpoints.tabletWide })} {
-    padding: ${spacing.xsmall} ${spacing.nsmall};
-  }
-`;
+const ContentWrapper = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+});
 
-const StyledImage = styled.img`
-  height: 150px;
-  width: 150px;
-`;
+const StyledImage = styled("img", {
+  base: {
+    height: "surface.3xsmall",
+    width: "surface.3xsmall",
+  },
+});
 
-const TitleWrapper = styled.div`
-  color: ${colors.text.primary};
-  font-family: ${fonts.sans};
-  font-weight: ${fonts.weight.bold};
-  overflow-wrap: break-word;
-  ${fonts.sizes("38px", "48px")};
-  text-align: center;
+const TitleWrapper = styled("div", {
+  base: {
+    fontSize: "4xlarge",
```

Comment:
Finnes det ikke en textstyle som kan brukes her?

Reply:
Hvordan finner man ut det?
Jeg syns det er litt convoluted å navigere disse tingene.

Reply:
Tilgjengelige valg? Typography-fil/story eller figma. Burde dukke opp med autocomplete

Reply:
Er vi forresten sikre på at vi vil det?
Iom at vi støtter html som input her kan vi ikke potensielt fakke med litt semantikk om vi evt gjør det?

Reply:
> Tilgjengelige valg? Typography-fil/story eller figma. Burde dukke opp med autocomplete

Åja, det er en panda greie.
Tenkte på denne `Text` komponenten vår jeg.

Ja autocompleten er jo der, men må fortsatt til fila for å vite hva jeg får.